README.md

```diff
@@ -0,0 +1,4 @@
+document-generator
```

Comment:
This should be prefixed with a '#' to format it as a title

Reply:
Have added